src/test/unit/controls/Scatter/ScatterLoad-spec.js

```diff
@@ -511,10 +509,10 @@ describe("Scatter - Load", () => {
                     );
                     expect(scatterGroup.length).toBe(1);
                     expect(
-                        scatterGraphContainer
-                            .querySelectorAll(`.${styles.point}`)[2]
-                            .getAttribute("aria-hidden")
-                    ).toBe("true");
+                        scatterGraphContainer.querySelectorAll(
+                            `.${styles.point}`
```

Comment:
Why is this impacting current tests?

Reply:
The current tests were checking whether a point exists in the DOM whose y value is null and having `aria-hidden` to be `true`. But after the changes that point is not present in the DOM itself, So the tests were failing. 
In the tests now, it is checking the total no. of `carbon-data-points` present in the DOM. Which is then compared with the length of dataArray after subtracting the no of data points with y value null.